<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>orderDataRecord</class>
 <widget class="QWidget" name="orderDataRecord">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>809</width>
    <height>77</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">#widget, #lblCategory, #lblId, #lblName, #lblRate, #lblamt{
	background-color: #ffffff;
	border-radius:5px;
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>10</number>
      </property>
      <item>
       <widget class="QLabel" name="lblId">
        <property name="text">
         <string>id</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblName">
        <property name="text">
         <string>name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblCategory">
        <property name="text">
         <string>category</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblRate">
        <property name="text">
         <string>rate</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblQty">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>qty</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
